qiita_pet/handlers/util.py

```diff
@@ -64,3 +64,28 @@ def download_link_or_path(is_local_request, filepath, fp_id, label):
 
 doi_linkifier = partial(
     linkify, "<a target=\"_blank\" href=\"http://dx.doi.org/{0}\">{0}</a>")
+
+
+def to_int(value):
+    """Transforms `value` to an integer
+
+    Parameters
+    ----------
+    value : str or int
+        The value to transform
+
+    Returns
+    -------
+    int
+        `value` as an integer
+
+    Raises
+    ------
+    HTTPError
+        If `value` cannot be transformed to an integer
+    """
+    try:
+        res = int(value)
+    except ValueError:
+        raise HTTPError(500, "%s cannot be converted to an integer" % value)
```

Comment:
Not sure if this is something we want to be specific about or simply say: "Not valid study identifier"